pkg/controllers/automigration/controller.go

```diff
@@ -327,14 +338,20 @@ func (c *Controller) estimateCapacity(
 
 		unschedulable, nextCrossIn := countUnschedulablePods(pods, time.Now(), unschedulableThreshold)
 		keyedLogger.V(3).Info("Analyzed pods",
```

Comment:
Should we print this at V(1) if there are unschedulable pods?

Reply:
V(2) seems more appropriate